packages/core/lib/src/widgets/user/username.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+
+/// Twitter user's username (handle)
+///
+/// e.g. @widgetbook_io
+class Username extends StatelessWidget {
+  /// Creates a new instance of [Username]
+  const Username({
+    super.key,
+    required this.value,
+    this.color,
+  });
+
+  /// Username value
+  final String value;
```

Comment:
If the username is `widgetbook_io` the widget currently shows `widgetbook_io` without the `@` sign. Therefore, I think the comment is a bit confusing.

Reply:
Agreed. Fixed this.